src/app/main/page.tsx

```
@@ -25,25 +25,32 @@ async function getBlockChildren(blockId: string) {
   }
 }
 export default async function Main() {
- 
   const data = await getNotionData();
   const scheme_text: string[] = [];
+  const preview_image: string[] = [];
   for (let item of data) {
```

ESLint

[error] 'item' is never reassigned. Use 'const' instead.

(prefer-const)